app/components/filters/exclude_cancelled.rb

```diff
@@ -0,0 +1,9 @@
+module Filters
```

Comment:
I'm not sure about the namespace hierarchy here. We've got a top level namespace `Filters` that internally instantiates an `ETL::Filter`. Maybe this object belongs inside the `ETL` namespace?